dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -18,9 +65,27 @@ namespace Microsoft.Windows.AppNotifications.Builder
         Boolean IncomingCallAlignment;
         Int32 MaxLines;
 
+        [contract(AppNotificationBuilderContract, 2)]
+        AppNotificationTextStyle Style;
+        [contract(AppNotificationBuilderContract, 2)]
+        Boolean Wrap;
```

Comment:
What is this for? 😅

Reply:
Will remove: the doc says: This property is not used. Setting this has no effect. If you need to disable wrapping, set hintMaxLines to 1.

Reply:
Removed, thanks for spotting this